internal/testidentity/pregenerated_identities_test.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testidentity
+
+import (
+	"crypto/x509"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/identity"
+	"storj.io/storj/pkg/peertls"
+	"storj.io/storj/pkg/storj"
+)
+
+func TestPregeneratedIdentity(t *testing.T) {
+	IdentityVersionsTest(t, func(t *testing.T, version storj.IDVersion, ident *identity.FullIdentity) {
+		assert.Equal(t, version.Number, ident.ID.Version().Number)
```

Comment:
I think in this case it makes sense to continue checking other assertions if the version isn't equal. I would prefer to know about multiple failed assertions when running the test rather than having to find out one at a time as I fix a failing test. Because the version being wrong doesn't cause a panic or an unexpected error I think it's reasonable to allow the test to continue.